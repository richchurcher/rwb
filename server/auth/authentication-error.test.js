import AuthenticationError from './authentication-error'

test('AuthenticationError matches the last snapshot', () => {
  expect(new AuthenticationError('Uh-oh', 403)).toMatchSnapshot()
})
