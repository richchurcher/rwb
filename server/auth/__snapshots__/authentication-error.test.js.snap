// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthenticationError matches the last snapshot 1`] = `[AuthenticationError: Uh-oh]`;
